modules/compliance-operator-cli-installation.adoc

```diff
@@ -28,15 +23,16 @@ metadata:
     openshift.io/cluster-monitoring: "true"
   name: openshift-compliance
 ----
-
-. Create the `OperatorGroup` object YAML file by running:
+. Create the `Namespace` object:
 +
 [source,terminal]
 ----
-$ oc create -f <file-name>.yaml
+$ oc create -f namespace-object.yaml
```

Comment:
I like the `<file-name>.yaml` structure as it indicates the user can use any name. But I think an admin user can follow this procedure.

Reply:
Ok, ty!- I will leave it this way for now - it was patterned after this [example](https://docs.openshift.com/container-platform/4.9/authentication/ldap-syncing.html#ldap-auto-syncing_ldap-syncing-groups).